docs/05_operations/08_update-queue/04_jms.mdx

```diff
@@ -0,0 +1,133 @@
+---
+title: 'Update queue - JMS'
+sidebar_label: 'JMS'
+id: jms
+keywords: [operations, update queue, jms, jakarta messaging]
+tags:
+- operations
+- update queue
+- jms
+- jakarta messaging
+---
+
+The Genesis low-code platform offers the flexibility to use a [Jakarta Messaging](https://en.wikipedia.org/wiki/Jakarta_Messaging) (former JMS) compliant message broker as its real-time update queue backend via [ArtemisMQ](https://activemq.apache.org/components/artemis/).
+
+Using a centralised external broker is highly recommended for complex, large or dynamically scaled clusters, as it reduces the complexity and overhead of peer-to-peer connectivity at the cost of maintaining or (re-using) a message broker.
+
+## Prerequisites
+
+Configuring Genesis to use Jakarta Messaging requires having **ArtemisMQ installed in the host** (or hosts in case of HA - see details [below](#high-avaiability)) as per [ArtemisMQ documentation](https://activemq.apache.org/components/artemis/documentation/).
```

Comment:
We want the message here to be very clear that installing the broker is not our problem (for now).